package-lock.json

```diff
@@ -2118,7 +2118,7 @@
     },
     "events": {
       "version": "1.1.1",
-      "resolved": "https://registry.npmjs.org/events/-/events-1.1.1.tgz",
+      "resolved": "http://registry.npmjs.org/events/-/events-1.1.1.tgz",
```

Comment:
http?

Reply:
You can remove package-lock.json, and node_module folder, and try to install again then it should be https

Reply:
Seems not working..